### PR TITLE
DevCommunity Summit 2021

### PR DESCRIPTION
**Pull request by handstandsam**

### CODE REVIEW

File: _conferences/devcommunity-summit-2021.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+name: "DevCommunity Summit 2021"
+website: https://twitter.com/devcommunityorg
```
Comment (cortinico) [inline _conferences/devcommunity-summit-2021.md]:
```suggestion
website: https://devcommunity.org/
```

File: _conferences/devcommunity-summit-2021.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+name: "DevCommunity Summit 2021"
```
Comment (cortinico) [inline _conferences/devcommunity-summit-2021.md]: Do you want to mention "Previously Android Summit" in the name as well?

Comment (handstandsam) [reply _conferences/devcommunity-summit-2021.md]: Good call, will update!

**Comment by handstandsam** [comment]
@cortinico - Made the suggested updates, and also adjusted the CFP close date.  We pushed it back 2 weeks.

Thank you!

**Pull request merged**